Fortinet/fortigate/ingest/parser.yml

```
@@ -0,0 +1,230 @@
+name: fortigate
+pipeline:
+  - name: parsed_event
+    external:
+      name: cef.parse-cef
+    filter: '{{original.message.startswith("CEF")}}'
+
+  - name: parsed_event
+    external:
+      name: kv.parse-kv
+      properties:
+        value_sep: '='
+        item_sep: ','
+    filter: "{{original.message | re_match('^\\s*\\w+=[^,]+(?:,\\w+=.*)\\s*$')}}"
+
+  - name: parsed_event
+    external:
+      name: kv.parse-kv
+      properties:
+        value_sep: '='
+        item_sep: \s
+    filter: "{{original.message | re_match('^\\s*\\w+=[^\\s]+(?:\\s\\w+=.*)\\s*$')}}"
+
+  - name: field_extraction
+
+stages:
+  field_extraction:
+    actions:
+      - set:
+          action.name: "{{parsed_event.message.name or parsed_event.message.FTNTFGTaction or parsed_event.message.FortinetFortiGateaction or parsed_event.message.act or parsed_event.message.action}}"
+          action.outcome: "{{parsed_event.message.status or 'success'}}"
+          action.outcome_reason: "{{parsed_event.message.outcome_reason or parsed_event.message.msg.split('\n')[0].strip('\"')}}"
```

```suggestion
          action.outcome_reason: "{{parsed_event.message.outcome_reason or parsed_event.message.msg.split('\n')[0].strip('\"')}}"
```

replaced by `event.reason`